src/compiler/checker.ts

```diff
@@ -16024,9 +16024,37 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
         // where different generations of the same type parameter are in scope). This leads to a lot of new type
         // identities, and potentially a lot of work comparing those identities, so here we create an instantiation
         // that uses the original type identities for all unconstrained type parameters.
+        //

```

Comment:
Maybe just create a break in the comment so it's clear the new stuff refers to the code that immediately follows.

```suggestion

```

Reply:
I changed the comment location, I hope this helps.